src/components/Dashboard/widgets/TokenDetailWidget.tsx

```diff
@@ -138,16 +138,20 @@ const MetadataItem = ({ name, value, copyValue }: MetadataItemProps) => (
   </MetadataItemRoot>
 );
 
+// https://datatracker.ietf.org/doc/html/rfc6838
+const mediaTypes = ['audio', 'video', 'image', 'ipfs'];
+
 const TokenDetail = () => {
   const tokenDetail = useSelector(selectCurrentTokenDetail);
   const [imageUrl, setImageUrl] = useState(null);
+  const isMedia = tokenDetail.contentType && mediaTypes.includes(tokenDetail.contentType);
```

Comment:
You can coerce the left side into a Boolean here: `!!tokenDetail.contentType &&`